deploy/examples/create-external-cluster-resources.py

```diff
@@ -58,9 +58,11 @@
 try:
     # for 2.7.x
     from urlparse import urlparse
+    from urllib import urlencode as urlencode
 except ModuleNotFoundError:
     # for 3.x
     from urllib.parse import urlparse
+    from urllib.parse import urlencode as urlencode
 
```

Comment:
there are multiple spelling issues, maybe fix them by adding another commit

Reply:
@parth-gr yeah, I fixed them by another PR: https://github.com/rook/rook/pull/12988 .. I'll rebase on it singe it's already merged.